locale/zh_CN/LC_MESSAGES/clr-installer.po

```diff
@@ -812,3 +812,16 @@ msgstr "需要网络"
 
 msgid "Installation Steps Complete"
 msgstr "安装步骤完成"
+
+# cracklib messages
+msgid "it does not contain enough DIFFERENT characters"
+msgstr "密码不包含足够的不同字符"
+
+msgid "it is too simplistic/systematic"
+msgstr "密码太简单"
+
+msgid "it is based on a dictionary word"
+msgstr "密码基于词典词"
+
+msgid "it is based on a (reversed) dictionary word"
+msgstr "La contraseña se basa en una palabra del diccionario invertida"
```

Comment:
This needs to be updated for the correct locale

Reply:
Sorry about that..done